generation_config.yaml

```diff
@@ -1081,6 +1081,21 @@ libraries:
   GAPICs:
   - proto_path: google/iam/credentials/v1
 
+- api_shortname: iap
+  name_pretty: Cloud Identity-Aware Proxy API
+  product_documentation: https://cloud.google.com/iap
+  api_description: Controls access to cloud applications running on Google Cloud Platform.
+  client_documentation: https://cloud.google.com/java/docs/reference/google-cloud-iap/latest/overview
+  release_level: stable
+  distribution_name: com.google.cloud:google-cloud-iap
+  api_id: iap.googleapis.com
+  library_type: GAPIC_AUTO
+  group_id: com.google.cloud
+  cloud_api: true
+  GAPICs:
+  - proto_path: google/cloud/iap/v1
+  - proto_path: google/cloud/iap/v1beta1
```

Comment:
Did you manually add v1beta1 here? I don't think we still have to generate v1beta1 since v1 is already here. [Dotnet](https://github.com/googleapis/google-cloud-dotnet/tree/main/apis/Google.Cloud.Iap.V1) also don't have it generated.